### DISCUSSION TITLE
Reverse the planned route

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#34] [Q&A] [answered]
It would be very practical if you could simply reverse the planned route.

**Comment by devemux86** [comment, marked as answer]
"Reverse route" is available in the route actions, when **long press on the route**:

<img src="https://user-images.githubusercontent.com/3484020/218272399-a4e35403-6f36-4e08-bcb8-baf5dd0a77a9.jpg" width="50%"/>

**Reply by Leeb64** [reply]
Sometimes you can't see the trees for the forest......
